zipline/finance/performance/tracker.py

```diff
@@ -267,7 +267,8 @@ def to_dict(self, emission_type=None):
     def process_event(self, event):
         self.event_count += 1
 
-        if event.type == zp.DATASOURCE_TYPE.TRADE:
+        if event.type in (
+                zp.DATASOURCE_TYPE.TRADE, zp.DATASOURCE_TYPE.LIQUIDATION):
```

Comment:
Could be negligible overall, but this is part of the tight loop so shaving nanoseconds can add up.

Defining the allowed values outside of the loop gets the comparison back to equivalent (or better) than the straight `==` check.

```
In [1]:
class Foo(object):
    ONE = 1
    TWO = 2
    THREE = 3

In [2]:
%%timeit

1 == Foo.ONE
10000000 loops, best of 3: 52.6 ns per loop


In [3]:
%%timeit

1 in (Foo.ONE, Foo.TWO)
10000000 loops, best of 3: 121 ns per loop


In [4]:
ALLOWED = (Foo.ONE, Foo.TWO)

In [5]:
%%timeit

1 in ALLOWED
10000000 loops, best of 3: 44.1 ns per loop
```